### ISSUE TITLE
global $ost not defined when $plugin->bootstrap()

### ISSUE THREAD

**Issue by quazardous** [osTicket/osTicket#2902]
Hi,

can you just add one line to allow plugins to access $ost ?

``` php
class osTicket {
...
    function start() {
        global $ost; // <--- this line
        // Prep basic translation support
        Internationalization::bootstrap();

        if(!($ost = new osTicket()))
            return null;

        //Set default time zone... user/staff settting will override it (on login).
        $_SESSION['TZ_OFFSET'] = $ost->getConfig()->getTZoffset();
        $_SESSION['TZ_DST'] = $ost->getConfig()->observeDaylightSaving();

        // Bootstrap installed plugins
        $ost->plugins->bootstrap();

        // Mirror content updates to the search backend
        $ost->searcher = new SearchInterface();

        return $ost;
    }
}


```

**Comment by quazardous** [comment]
i think it can fix #1074

You can inject headers...
```php
...
$ost->addExtraHeader('<!-- BOUYA -->');
...
```

**Comment by fnbf** [comment]
Ping.
